src/NserviceBus.Transport.Msmq/MsmqTimeToBeReceivedOverrideCheck.cs

```diff
@@ -19,8 +19,19 @@ public StartupCheckResult CheckTimeToBeReceivedOverrides()
             var isTransactional = settings.GetRequiredTransactionModeForReceives() != TransportTransactionMode.None;
             var outBoxRunning = settings.IsFeatureActive(typeof(Features.Outbox));
 
-            var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
-            var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
+            //var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //workaround for AuditConfigReader.Result not being accessible
```

Comment:
@andreasohlund - since there's limitations around TTBR for audit messages, do we even care? Should we get rid of the TTBR option for audit messages? 
Also as discussed with @bording, I think the AuditConfigReader will be so much easier after getting rid of the registry setting as well.